src/plugins/faultManagement/FaultManagementListView.vue

```diff
@@ -87,32 +110,31 @@ export default {
             list.sort(SORT_ITEMS[this.sortBy].sortFunction);
 
             return list;
+        },
+        showToolbar() {
+            const faultActionProvider = this.openmct.faults.faultActionProvider;
```

Comment:
This should be a method to check whether fault actions are supported. The provider itself is a detail we shouldn't expose to the UI.